src/xcode/ENA/ENA/Source/Workers/Store/Store.swift

```diff
@@ -60,6 +60,8 @@ protocol StoreProtocol: AnyObject {
 
 	var riskCalculationResult: RiskCalculationResult? { get set }
 
+	var dateOfConversionToHighRisk: Date? { get set }
```

Comment:
if this is with ppa at that place, so it should be moved from here to ppa section below.